go/stellar/build.go

```diff
@@ -386,6 +447,67 @@ func ReviewPaymentLocal(mctx libkb.MetaContext, stellarUI stellar1.UiInterface,
 	return mctx.Ctx().Err()
 }
 
+// identifyForReview runs identify on a user, looking only for tracking breaks.
+// Sends a value to exactly one of the three channels.
+func identifyForReview(mctx libkb.MetaContext, assertion string,
+	successCh chan<- struct{},
+	trackFailCh chan<- struct{},
+	errCh chan<- error) {
+	// Goroutines that are blocked on otherwise unreachable channels are not GC'd.
+	// So use ctx to clean up.
+	sendSuccess := func() {
+		mctx.CDebugf("identifyForReview(%v) -> success", assertion)
+		select {
+		case successCh <- struct{}{}:
+		case <-mctx.Ctx().Done():
+		}
+	}
+	sendTrackFail := func() {
+		mctx.CDebugf("identifyForReview(%v) -> fail", assertion)
+		select {
+		case trackFailCh <- struct{}{}:
+		case <-mctx.Ctx().Done():
+		}
+	}
+	sendErr := func(err error) {
+		mctx.CDebugf("identifyForReview(%v) -> err %v", assertion, err)
+		select {
+		case errCh <- err:
+		case <-mctx.Ctx().Done():
+		}
+	}
+
+	mctx.CDebugf("identifyForReview(%v)", assertion)
+	reason := fmt.Sprintf("Identify transaction recipient: %s", assertion)
+	eng := engine.NewResolveThenIdentify2(mctx.G(), &keybase1.Identify2Arg{
+		UserAssertion:         assertion,
+		CanSuppressUI:         true,
+		NoErrorOnTrackFailure: true, // take heed
+		Reason:                keybase1.IdentifyReason{Reason: reason},
+		IdentifyBehavior:      keybase1.TLFIdentifyBehavior_RESOLVE_AND_CHECK,
```

Comment:
There are so many to choose from. Is this a good one? I just need track breaks to be part of the result, not error. And no tracker popups should pop up. This seems to unfortunately disable the cache too. @maxtaco

Reply:
I think this is good?